starter/angular-xeditable/js/xeditable.js

```diff
@@ -1058,6 +1058,7 @@ angular.module('xeditable').factory('editableFormController',
       if (this.$visible) {
         editable.catchError(editable.show());
       }
+      editable.catchError(editable.setWaiting(this.$waiting));
```

Comment:
Have you pulled on a different branch? This change (i don't think) is meant to be in this pull req.